helmfile/overrides/system/fluentbit-agent.yaml.gotmpl

```diff
@@ -0,0 +1,401 @@
+env:
```

Comment:
I remember we had some difficulties with fluentbit back when we enabled it with @ben851 using kustomize. Is that a copy of the same configuration? 👀

Reply:
I'll take this up on a. slack DM chat, but the quick answer is that it's the same configurations, but they've been completely migrated and modified where necessary to be a helmfile overrides file